src/main/java/dorkix/armored/elytra/ArmoredElytra.java

```diff
@@ -112,18 +117,46 @@ public static ItemStack createArmoredElytra(ItemStack elytra, ItemStack armor,
 
 		// Set description
 		var armorHasCustomName = armor.get(DataComponentTypes.CUSTOM_NAME) != null;
+
+		List<Text> loreTexts = Lists.newArrayList();
+
+		if (trims != null && trims.isPresent() && loreTexts != null) {
+			List<Text> trimTexts = Lists.newArrayList();
+			// We cannot add the trim component to the armored elytra bacause of the
```

Comment:
Nice catch! I just thought of this and came back to add it, only to find you already got to it! =D